research/src/pages/tabs.proposal.mdx

```diff
@@ -0,0 +1,283 @@
+
+### Background <a href="#background" id="background"></a>
+
+_Relevant historical or background information, related existing issues, etc._
+
+### Use Cases <a href="#use-cases" id="use-cases"></a>
+
+- Steve is researching for an upcoming vacation for him and his family. He opens his web browser and opens a new tab to make a rental car reservation and another for his flight information. Steve can quickly switch back and fourth between tabs to get the information he needs for the rental car reservation.
+
+- Monika visits a website to learn how to fix her leaky sink. The website displays a vertical tab interface with each step. Monika can jump ahead steps or go back to previous steps by selecting various tabs.
+
+### Non-goals <a href="#non-goals" id="non-goals"></a>
+
+_A list of use cases, features, or functionality which are **not** goals for the component._
+
+### Features <a href="#features" id="features"></a>
+
+- **Orientation:** Allows the tab list to be oriented horizontally above the tab content or vertically to the left or right (depending on language region) of the tab content.
+- **Supplemental content:** Offers a way to add content via start and end to the left and/or right (depending on language region) of the the tab list.
+- **ActiveIndicator:** Offers a way to add or remove an active indicator that highlights the currently active tab and animates to the next active tab. Defaults to true.
+- **ActiveTab:** Provides a reference to the currently active tab vis the `change` event.
+- **ActiveId:** Provides a way to set the active tab.
+
+### Risks and Challenges <a href="#risks" id="risks"></a>
+
+*Tabs* has specific guidance on DOM structure by the [WC3](https://w3c.github.io/aria-practices/examples/tabs/tabs-2/tabs.html). This structure lacks logical groupings by separating the tab from its content. Some component libraries compensate for this by creating some kind of intermediary grouping that makes it easier for app authors to implement. Even though the DOM structure disassociates that logical grouping. While other component libraries stick to the DOM structure model. 
+
+Some scenarios require an indicator that highlights the currently active tab then animates to the next activated tab. Most solutions rely on finding active tab's position on screen and then preforming some math to get the position. This works for most cases but if the tab list repositions itself either by window resizing or other layout changes the active indicator is no longer aligned properly.
+
+### Prior Art/Examples <a href="#prior-art" id="prior-art"></a>
+
+- [FAST Tabs](https://github.com/microsoft/fast/blob/master/packages/web-components/fast-foundation/src/tabs/tabs.spec.md)
+- [FAST Pivot (React)](https://www.npmjs.com/package/@microsoft/fast-components-react-msft)
+- [Material UI](https://material-ui.com/components/tabs/)
```

Comment:
I would replace this with the official Material Design docs (https://material.io/components/tabs#anatomy)

Reply:
I think it might be good to have both. Eventually this section will have many popular tabs implementations that we can reference to talk about how these solve the various challenges. So maybe splitting this into Design Systems and Tabs Implementations makes sense?